rudder-rest/src/main/scala/com/normation/rudder/rest/lift/SystemApi.scala

```diff
@@ -27,4 +38,99 @@ class SystemApi(restExtractorService: RestExtractorService) extends LiftApiModul
       toJsonResponse(None,("global" -> "ok"))
     }
   }
+
+  object TechniquesReload extends LiftApiModule0 {
+    val schema = API.TechniquesReload
+    val restExtractor = restExtractorService
+
+    def process0(version: ApiVersion, path: ApiPath, req: Req, params: DefaultParams, authzToken: AuthzToken): LiftResponse = {
+      apiv11service.reloadTechniques(req, params)
+    }
+  }
+
+  object DyngroupsReload extends LiftApiModule0 {
+    val schema = API.DyngroupsReload
+    val restExtractor = restExtractorService
+
+    def process0(version: ApiVersion, path: ApiPath, req: Req, params: DefaultParams, authzToken: AuthzToken): LiftResponse = {
+      apiv11service.reloadDyngroups(params)
+    }
+  }
+
+  object ReloadAll extends LiftApiModule0 {
+    val schema = API.ReloadAll
+    val restExtractor = restExtractorService
+
+    def process0(version: ApiVersion, path: ApiPath, req: Req, params: DefaultParams, authzToken: AuthzToken): LiftResponse = {
+      apiv11service.reloadAll(req, params)
+
+    }
+  }
+}
+
+class SystemApiService11(
+    updatePTLibService      : UpdateTechniqueLibrary
+  , uuidGen                 : StringUuidGenerator
+  , updateDynamicGroups     : UpdateDynamicGroups
+  , implicit val userService: UserService
+) extends Loggable {
+
+  private[this] def reloadTechniquesWrapper(req: Req) : Either[String, JField] = {
+
+    updatePTLibService.update(ModificationId(uuidGen.newUuid), getActor(req), Some("Technique library reloaded from REST API")) match {
+      case Full(x) => Right(JField("techniques", "Reloaded"))
```

Comment:
I think the answer should be the same in both enpoints, Put reloaded for now in both

Reply:
We should discuss it elsewhere on slack